### ISSUE TITLE
Asking for an advise

### ISSUE THREAD

**Issue by naeunang** [MahmudulAlam/Complete-Blood-Cell-Count-Dataset#3]
Hello! Thank you for your great work. I have a few questions.

I am trying to train a model with dental panoramic radiographs. I have similar number of dataset as you did. However, I have much smaller number of labels. It means that I have about 1000 datasets with about 1-3 labels in each image. I am wondering if this is why I keep getting decreasing training error with increasing validation error due to an overfitting. I know that I do not have enough samples and overfitting occurs naturally but it happens too early. Also, training and validation sets are too different in performance. 

Would I be able to know your cfg settings? Have you had this kind of problem? Would you give me an advise if you did? We have used yolov4 in darknet and had the same problem using tiny. Please give us some advice if you could. 

Thank you for reading this comment and I hope you have a great day!

**Comment by MahmudulAlam** [comment]
@naeunang 
Hi, sorry for the late response. Try the following to overcome the overfitting problem: 
1. Try regularization 
2. Data augmentation 
you can open the cfg files using any text editor. A certain percentage of overfitting is ok IMO, often times validation accuracy is lower than the training accuracy. 10-20% difference is quite reasonable.
